src/Domain/RegularGivingMandate.php

```diff
@@ -247,7 +248,7 @@ public function getUuid(): UuidInterface
 
     public function getAmount(): Money
```

Comment:
worth also renaming this function. PHPStorm should be able to rename it and update all usages at once.

Reply:
I tried, but it's overridden method
![2024-12-17_13-09](https://github.com/user-attachments/assets/57fccf54-92f2-4f0a-87cf-39e7f7745ac3)